tests/AbstractTestCase.php

```diff
@@ -10,7 +10,6 @@
 use Brick\DateTime\LocalDateRange;
 use Brick\DateTime\LocalDateTime;
 use Brick\DateTime\LocalTime;
-use Brick\DateTime\Month;
```

Comment:
How is this not caught by ECS? :thinking:

Reply:
Good question... I've had my share of unused imports not being picked up by ECS, but I've never understood how it comes to be. Never bothered to open an issue, as I'm working on a private project for which I can't just share the whole codebase to help reproduce it.

But maybe it would make sense to create a fork of brick/date-time and share it with @tomasvotruba so he could investigate.

Reply:
There it is : https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7908